question19/js/newsTopics.js

```diff
@@ -0,0 +1,159 @@
+'use strict';
+import { differenceInDays } from "date-fns";
+
+const createElementWithClassName = (element, name) => {
+  const createdElement = document.createElement(element);
+  createdElement.classList.add(name);
+  return createdElement;
+};
+
+const newsWrapper = document.getElementById("js-newsWrapper");
+const tabsGroup = document.getElementById("js-tabs");
+const contentsWrapper = createElementWithClassName("div", "contentsWrapper");
+contentsWrapper.id = "js-contentsWrapper";
+
+//Get json data
+async function callApi() {
+  try {
+    const res = await fetch(
+      "https://mocki.io/v1/5a30de87-dfe7-4644-a197-b832f1937adb"
+    );
+    if (!res.ok) {
+      throw new Error(`サーバーリクエストに失敗しました: ${res.status}`);
+    }
+    const json = await res.json();
+    return json.data;
+  } catch (error) {
+    console.error(error);
+  } finally {
+    console.log("CONPLETE!");
+  }
```

Comment:
Q.
Why is `console.log("CONPLETE!");` necessary?
Please tell us!

Reply:
Thank you for your review! The previous implementation remained in place. It is not needed and has been removed. Thanks for pointing that out!
fixed commit : https://github.com/csakyo/homework/pull/33/commits/06f7e0d2e85844b9b9cacfd2b6658cee88d6d98f